atuin/src/command/client/search/interactive.rs

```diff
@@ -606,8 +606,13 @@ impl State {
         stats
     }
 
-    fn build_results_list(style: StyleState, results: &[History]) -> HistoryList {
-        let results_list = HistoryList::new(results, style.invert);
+    fn build_results_list<'a>(
+        style: StyleState,
+        results: &'a [History],
+        keymap_mode: KeymapMode,
+    ) -> HistoryList<'a> {
```

Comment:
```suggestion
    fn build_results_list(
        style: StyleState,
        results: &[History],
        keymap_mode: KeymapMode,
    ) -> HistoryList<'_> {
```

The lifetime here can be elided

Reply:
Fixed it, left in the `<'_>` elided lifetime marker as would be required by the `elided_lifetimes_in_paths` lint as it makes it obvious that there's a lifetime dependency here.